unit-tests/run-unit-tests.py

```diff
@@ -11,56 +11,61 @@
 # otherwise pyrs other than what we compiled might be found...
 sys.path = list()
 sys.path.append( '' )  # directs Python to search modules in the current directory first
-sys.path.append( os.path.dirname( sys.executable ))
-sys.path.append( os.path.join( os.path.dirname( sys.executable ), 'DLLs' ))
-sys.path.append( os.path.join( os.path.dirname( sys.executable ), 'lib' ))
+sys.path.append( os.path.dirname( sys.executable ) )
+sys.path.append( os.path.join( os.path.dirname( sys.executable ), 'DLLs' ) )
+sys.path.append( os.path.join( os.path.dirname( sys.executable ), 'lib' ) )
 # Add our py/ module directory
-current_dir = os.path.dirname( os.path.abspath( __file__ ))
+current_dir = os.path.dirname( os.path.abspath( __file__ ) )
 sys.path.append( current_dir + os.sep + "py" )
 
 from rspy import log, file, repo
 
+
 def usage():
-    ourname = os.path.basename(sys.argv[0])
+    ourname = os.path.basename( sys.argv[0] )
     print( 'Syntax: ' + ourname + ' [options] [dir]' )
-    print( '        dir: the directory holding the executable tests to run (default to the build directory')
+    print( '        dir: the directory holding the executable tests to run (default to the build directory' )
     print( 'Options:' )
     print( '        --debug        Turn on debugging information' )
     print( '        -v, --verbose  Errors will dump the log to stdout' )
     print( '        -q, --quiet    Suppress output; rely on exit status (0=no failures)' )
     print( '        -r, --regex    run all tests that fit the following regular expression' )
     print( '        -s, --stdout   do not redirect stdout to logs' )
     print( '        -t, --tag      run all tests with the following tag' )
-    print( '        --list-tags    print out all available tags. This option will not run any tests')
-    print( '        --list-tests   print out all available tests. This option will not run any tests')
-    sys.exit(2)
+    print( '        tests automatically get tagged with \'exe\' or \'py\' and based on their location inside unit-tests/,' )

```

Comment:
Indent to align with `run` on prev line.
Split line up as needed to make readable on 80-char terminal.
Please add description indicating how to combine multiple tags.